packages/website/src/components/editor/config.ts

```diff
@@ -6,14 +6,21 @@ export function createCompilerOptions(
   jsx = false,
   tsConfig: Record<string, unknown> = {},
 ): Monaco.languages.typescript.CompilerOptions {
-  return {
-    noResolve: true,
+  const config: Monaco.languages.typescript.CompilerOptions = {
     // ts and monaco has different type as monaco types are not changing base on ts version
     target: window.ts.ScriptTarget.ESNext as number,
     module: window.ts.ModuleKind.ESNext as number,
     ...tsConfig,
     jsx: jsx ? window.ts.JsxEmit.Preserve : window.ts.JsxEmit.None,
   };
+
+  const libs = Array.isArray(tsConfig.lib) ? tsConfig.lib : ['es6', 'dom'];
```

Comment:
this is not fully correct, we should deduce libs if not provided from target and module :(